derive_builder/src/options/mod.rs

```diff
@@ -233,14 +274,12 @@ impl<Mode> OptionsBuilder<Mode> where
             match *x {
                 syn::NestedMetaItem::MetaItem(ref meta_item) => {
                     self.parse_setter_options_metaItem(meta_item);
-                    // setters implicitly enabled
-                    if self.setter_enabled.is_none() {
-                        self.setter_enabled(true);
-                    }
                 },
-                syn::NestedMetaItem::Literal(ref _lit) => {
-                    // setters explicitly enabled
-                    self.setter_enabled(true);
+                syn::NestedMetaItem::Literal(ref lit) => {
+                    error!("Expected NestedMetaItem::MetaItem, found `{:?}`.", x);
```

Comment:
Is this user-visible by default? (I don't know what log-level is the default.) If so, I'd try to explain how that happens and how to fix it, not throw a "NestedMetaItem::MetaItem" in their face ;)